### PR TITLE
This is a tribute page of Nigerian President (President Muhammadu Buhari)

### PR DESCRIPTION
**Pull request by Dinma-Faith-zz**
This project is built with 
- HTML and CSS

### CODE REVIEW

**Review by Dinma-Faith-zz — COMMENTED** [review]
-Please make sure you have linters in your code as it checks for error and helps your code looks clean and easy to read.

-Also create an image folder and have your image in it. It would help your project be clean and professional.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
Hi @Dinma-Faith, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good 👍 mentioning about linter files setup.
- Good job 👍 for adding images in a folder.

## Required Changes ♻️
Changes which are important to mention are:
- Make sure to mention- a descriptive Pull Request to be added.
- Remember to mention- good commit messages are required.
- Kindly mention- A professional Readme is very important for the project overview so do implement it.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dinma-Faith-zz — COMMENTED** [review]
-Please make sure you have linters in your code as it checks for error and helps your code looks clean and easy to read.
-Create an image folder and have your image in it. It would help your project be clean and 
  professional.
- Please add a descriptive Pull Request. it gives a glimpse to different 
   stakeholders into what the PR is doing.
- Do well to also make a good commit message. It gives context to the code changes. 
- Also, professional Readme is very important for the project overview so do implement it.

- You can check [https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md](url)

Thanks

File: index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div id="main">
+      <!-- header -->
+      <div class="header-container myflex">
+        <div class="head-content">
+          <h1 id="title">President Buhari</h1>
+        </div>
+      </div>
```
Comment (aimemalaika) [inline index.html:17]: - [ ] [OPTIONAL] Consider using a `header` tag as it is the appropriate semantic tag for headers display

File: index.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        <div class="content">
+          <ul>
+            <p>
+              Muhammadu Buhari GCFR is a Nigerian politician who has been
+              President of Nigeria since 2015. Buhari is a retired Nigerian Army
+              Major General and has served as military head of state in Nigeria
+              between 1983 to 1985, after taking power in a military coup
+              d'état.
+            </p>
+            <li>
```
Comment (aimemalaika) [inline index.html:37]: - [ ] [OPTIONAL] The `p` tag is not an element of `ul`, consider taking it out of the `ul` tag

**Review by aimemalaika — COMMENTED** [review]
Hi @Dinma-Faith , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**